### ISSUE TITLE
App Hintergrundaktivität

### ISSUE THREAD

**Issue by kai-raschke** [sebsebli/infectiontracker#8]
Scheinbar hört die App im Hintergrund auf den Status abzufragen.
Wenn die App offen ist, wird weiter abgefragt.

Plattform: Android 10
Version: Prototyp 1

**Comment by sebsebli** [comment]
Ja. Das ist so. Meine Empfehlung ist hierfür Push! (Bei Änderungen Serverseitig)
